apps/esphome/tests.yaml

```diff
@@ -0,0 +1,13 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/goss-org/goss/master/docs/schema.yaml
+process:
+  esphome:
+    running: true
+
```

Comment:
Remove the empty lines here

Reply:
Done. Newlines are subjective AF though :P